certified-connectors/IN-D PayablesV3/apiDefinition.swagger.json

```diff
@@ -0,0 +1,485 @@
+                            "format": "float",
+                            "description": "The confidence score for vendor gstin extracted from the invoice",
+                            "title": "confidence score vendor gstin"
+                          }
+                        },
+                        "description": "vendor_gstin"
+                      },
+                      "customer_gstin": {
+                        "type": "object",
+                        "properties": {
+                          "value": {
+                            "type": "string",
+                            "description": "customer gstin extracted from the invoice",
+                            "title": "customer gstin value"
+                          },
+                          "conf_score": {
+                            "type": "number",
+                            "format": "float",
+                            "description": "The confidence score for customer gstin extracted from the invoice",
+                            "title": "confidence score customer gstin"
+                          }
+                        },
+                        "description": "customer_gstin"
+                      },
+                      "tabledata": {
+                        "type": "object",
+                        "properties": {
+                          "columns": {
+                            "type": "array",
+                            "items": {
+                              "type": "string"
+                            },
+                            "description": "The first row of the table is considered as the Header for each Column"
+                          },
+                          "data": {
+                            "type": "array",
+                            "items": {
+                              "type": "array",
+                              "items": {
+                                "type": "string"
+                              }
+                            },
+                            "description": "Each item corresponds to the values extracted in a row against the Header items"
+                          }
+                        },
+                        "description": "The line items in the Table is extracted"
+                      }
+                    }
+                  },
+                  "description": "The Result Data for the whole batch of invoices"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Invalid base64 string",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "message": {
+                  "type": "string",
+                  "description": "Invalid base64 string",
+                  "title": "message"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Invalid api key",
```

Comment:
Please check the meanings for 401 vs 403.
401 typically means auth key is not given or is expired. 403 means user supplied a valid key but the user lacks privileges to access the resource.

Reply:
Hello @samrat-gavale,
I have updated the swagger file and I have removed the 401 error code. because currently, we are not giving that error code.so, I have updated currently, what we have. can you please check now and approve my connector.